### ISSUE TITLE
About Dataset

### ISSUE THREAD

**Issue by cla7aye15I4nd** [zer0fall/BENZENE#4]
Dear authors:

I would like to inquire if it is possible to access the full dataset referenced in your paper.

**Comment by zer0fall** [comment]
The vulnerabilities used in our paper are accessible through public web (at least, at the moment of our evaluation)

So, you might be able to get required samples by visiting websites cited in our paper's "Reference" section.
If you have any trouble to get specific samples, please let me know.

And in case you face build issues of dataset, please note that we performed our evaluation (and compilation) on Ubuntu 20.04 LTS.

**Comment by cla7aye15I4nd** [comment]
I am particularly interested in reproducing the results of your studies. However, manually collecting all cases from the network could introduce significant bias, as the cases we gather might differ from those in your original dataset. I was wondering if you might have a backup of the dataset available. Access to it would be incredibly helpful for accurate replication and would also benefit others who may want to compare their tools with your work.

**Comment by zer0fall** [comment]
Unfortunately, we are not currently maintaining compiled binaries used at our evaluation 
(if I understood your question well).

Instead, we have compilation commands that we have actually used for our evaluation, 
so I think providing such build instructions for each dataset would help your work. 

So, I'll sum up the list, and share it to you.
(it might take some time because I'm currently assigned with quite urgent project)

Thank you!

**Comment by cla7aye15I4nd** [comment]
yes, I think compilation commands and poc is very enough, thanks for you help.

**Comment by merlinsun** [comment]
Dear @zer0fall,

I'm also very interested in this excellent work. I was wondering if the compilation commands and PoC used in the dataset are currently available, as I'd love to explore them further.
